dev/io.openliberty.jakarta.rest.4.0.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -0,0 +1,618 @@
+            <artifactId>io.openliberty.org.eclipse.parsson.1.1</artifactId>
+            <version>1</version>
+            <scope>system</scope>
+            <systemPath>${io.openliberty.org.eclipse.parsson.1.1_}</systemPath>
+        </dependency>        
+        <dependency>
+            <groupId>io.openliberty</groupId>
+            <artifactId>com.ibm.ws.org.apache.httpcomponents</artifactId>
+            <version>1</version>
+            <scope>system</scope>
+            <systemPath>${com.ibm.ws.org.apache.httpcomponents_}</systemPath>
+        </dependency>    
+        <dependency>
+            <groupId>io.openliberty</groupId>
+            <artifactId>com.ibm.ws.org.jboss.jandex</artifactId>
+            <version>1</version>
+            <scope>system</scope>
+            <systemPath>${com.ibm.ws.org.jboss.jandex_}</systemPath>
+        </dependency>    
+        <dependency>
+            <groupId>io.openliberty</groupId>
+            <artifactId>com.ibm.websphere.org.reactivestreams.reactive-streams.1.0</artifactId>
+            <version>1</version>
+            <scope>system</scope>
+            <systemPath>${com.ibm.websphere.org.reactivestreams.reactive-streams.1.0_}</systemPath>
+        </dependency>    
+    
+        <dependency>
+            <groupId>com.ibm.ws</groupId>
+            <artifactId>fattest.simplicity</artifactId>
+            <version>1</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/../../../lib/fattest.simplicity.jar</systemPath> <!-- This will point to build/libs/autoFVT/lib/fattest.simplicity.jar -->
+        </dependency>
+        <dependency> <!-- required by fattest.simplicity?? -->
+            <groupId>commons-httpclient</groupId>
+            <artifactId>commons-httpclient</artifactId>
+            <version>3.1</version>
+            <scope>test</scope>
+        </dependency>
+        <!--  Note that the JAX-RS dependency must be ahead of the Jakarta platform dependency until we move from EE9 to EE10. -->
+        <!--  This is required so that the JAX-RS 3.1 APIs are loaded first in the test classpath - ahead of the JAX-RS 3.0 APIs that come with EE9 platform. -->
+        <dependency>
+            <groupId>jakarta.ws.rs</groupId>
+            <artifactId>jakarta-restful-ws-tck</artifactId>
+            <version>${jakarta.ws.rs-tck.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.arquillian.junit5</groupId>
+            <artifactId>arquillian-junit5-container</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <!-- see: https://github.com/arquillian/arquillian-core/issues/248 -->
+        <!-- and https://github.com/arquillian/arquillian-core/pull/246/files -->
+        <dependency>
+            <groupId>org.jboss.arquillian.protocol</groupId>
+            <artifactId>arquillian-protocol-servlet-jakarta</artifactId>
```

Comment:
arquillian.xml has rest protocol.  Should this be rest-jakarta?

Reply:
what change are you suggesting?

Reply:
Either change arquillian-protocol-servlet-jakarta to arquillian-protocol.rest-jakarta or change arquillian.xml from rest to servlet